src/backend/distributed/utils/citus_depended_object.c

```diff
@@ -480,8 +480,10 @@ AnyObjectViolatesOwnership(DropStmt *dropStmt)
 		Node *object = NULL;
 		foreach_ptr(object, dropStmt->objects)
 		{
+			Relation rel;
```

Comment:
nit; we zero init everything

```suggestion
			Relation rel = NULL;
```